app/models/reservation_scopes.rb

```diff
@@ -1,51 +1,37 @@
 module ReservationScopes
   def self.included(base) # rubocop:disable MethodLength, AbcSize
     base.class_eval do
-      scope :recent, ->() { order('start_date, due_date, reserver_id') }
+      default_scope { order('start_date, due_date, reserver_id') }
       scope :user_sort, ->() { order('reserver_id') }
-      scope :finalized, lambda {
-        where('approval_status = ? OR approval_status = ?', 'auto',
-              'approved')
-      }
-      scope :not_returned, ->() { where('checked_in IS NULL').finalized }
-      scope :active, ->() { not_returned }
-      # TODO: grep codebase for active and replace with not_returned or
-      # figure out a better way to give the same scope two names
-      scope :untouched, ->() { where('checked_out IS NULL').not_returned }
-      scope :reserved, lambda {
-        where('due_date >= ?', Time.zone.today).untouched.recent
-      }
-      scope :checked_out, lambda {
-        where('checked_out IS NOT NULL').not_returned.recent
+
+      scope :not_available, lambda {
+        where(status: Reservation.statuses.values_at(
+          *%w(reserved checked_out)).compact)
       }
+      scope :active, ->() { not_available }
+      scope :finalized, ->() { not_available }
+
       scope :checked_out_today, lambda {
         where('checked_out >= ? and checked_out <= ?',
               Time.zone.today,
-              Time.zone.today + 1.day).not_returned.recent
+              Time.zone.today + 1.day).checked_out
```

Comment:
Do we want to restrict this scope to reservations that are still `checked_out`? What if a reservation was checked out today but has already been returned? This is more relevant for the `checked_out_previous` query, I guess.